sympy/stats/tests/test_continuous_rv.py

```diff
@@ -430,6 +430,7 @@ def test_gumbel():
     x = Symbol("x")
     X = Gumbel("x", beta, mu)
     assert simplify(density(X)(x)) == exp((beta*exp((mu - x)/beta) + mu - x)/beta)/beta
+    assert cdf(X)(x) == exp(-exp((-mu + x)/beta))
```

Comment:
The sign is still not correct. It should be `mu - x`, see the formula at https://en.wikipedia.org/wiki/Gumbel_distribution#Properties.

Reply:
@sidhantnagpal Corrected. Apologies for a lot of mistakes.